dev/com.ibm.ws.jaxrs.2.0.client/resources/com/ibm/ws/jaxrs20/client/internal/resources/JAXRSClientMessages.nlsprops

```diff
@@ -48,3 +48,11 @@ failed_to_extract_saml_token_from_subject.useraction=Ensure that the server.xml
 failed_run_as_subject=CWWKW0706E: An exception occurred while attempting to get the RunAsSubject. The exception was: [{0}].
 failed_run_as_subject.explanation=An exception occurred while getting the RunAsSubject.
 failed_run_as_subject.useraction=Make sure the server is configured and started properly.
+
+warn_missing_mpjwt_token=CWWKW0707W: The [{0}] attribute in the [{1}] configuration is set to [{2}], but the MicroProfile JSON Web Token(JWT) is not available. The request does not contain an Authorization header with the token.
+warn_missing_mpjwt_token.explanation=The runtime failed to find the MicroProfile JWT token in the runAs subject. This error usually indicates a problem with the authentication. Make sure that there are no errors encountered while authenticating a user.
+warn_missing_mpjwt_token.useraction=Check the user action for any prior errors that the server has logged.
```

Comment:
Our grammar checker also isn't a fan of perfect tense. Could we remove the "has" and change this to "Check the user action for any prior errors that the server logged." ?